.github/workflows/ruby-core.yml

```diff
@@ -36,7 +36,7 @@ jobs:
           path: ruby/ruby
           fetch-depth: 10
       - name: Checkout the latest buildable revision
-        run: git switch -c ${{ env.REF }}
+        run: git switch -c 57daed5ec5f12f64fd4891dc86d76b183c838e93
```

Comment:
revert?

Reply:
Oh, yeah, that was me debugging current red CI. I'll revert 👍.